db/schema.rb

```diff
@@ -162,7 +162,6 @@
     t.index ["role"], name: "index_users_on_role"
   end
 
-  add_foreign_key "activities", "activities"
```

Comment:
I am trying to understand why there are no foreign keys here and what the relationship is we want, I think it's about making sure the parent of a child is not deleted, orphaning the child  but I cannot see that reflected in the schema?

Reply:
I missed this removal and am adding it back in. I believe it is about making sure the parent of a child is not deleted.